ia/src/parser.py

```
@@ -0,0 +1,41 @@
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)